demo-app/src/main/java/com/gravatar/demoapp/ui/AvatarUpdateTab.kt

```diff
@@ -66,6 +68,9 @@ fun AvatarUpdateTab(modifier: Modifier = Modifier) {
     val scope = rememberCoroutineScope()
     var avatarUrl: String? by remember { mutableStateOf(null) }
     val keyboardController = LocalSoftwareKeyboardController.current
+    var pickerContentLayout: ContentLayout by rememberSaveable(stateSaver = ContentLayoutSaver) {
+        mutableStateOf(ContentLayout.Horizontal)
+    }
```

Comment:
This wouldn't be persisted on screen orientation so that would also cause a crash. This is a bit specific for our demo app I think, so I kept the Saver private.